app/assets/javascripts/comment.js

```diff
@@ -0,0 +1,9 @@
+$(function(){
```

Comment:
Hey @FaheemAhmedNoor ,
The `JavaScript` solution is good. But I'm wondering is there any `Ruby` method available to achieve the same thing or we need to do it in `JavaScript` and no `Ruby` way?